config.rb

```diff
@@ -77,13 +81,18 @@ def default_config
           'redact_ssn' => true,
           'cache_oidc_config' => true,
           'vtr_disabled' => ENV.fetch('vtr_disabled', 'false') == 'true',
+          'eipp_allowed' => false,
         }
 
         # EC2 deployment defaults
 
         env = ENV['idp_environment'] || 'int'
         domain = ENV['idp_domain'] || 'identitysandbox.gov'
 
+        if env == 'staging'
+          data['eipp_allowed'] = true
+        end
```

Comment:
IMO ideally we would rely on the `eipp_allowed` config being correct in staging, rather than checking against the env here

Reply:
@zachmargolis: @gina-yamada and I struggled with figuring out how to actually set the configuration value in staging. Could we maybe do a quick chat to see where it's set?

Reply:
I am not sure either. I am poking around.

Reply:
Ahhh cloud.gov. Got it. I wonder if we have access to the cloud.gov dashboard for this site? That should come along with github repo access, right?

Reply:
We may have to invite you to that separately, we can follow up in Slack

Reply:
I recently got access. I am looking around for the process. It looks like I can just add the new variable in staging and then  maybe/likely "restaging the app".  Looking into that.  (It recreates your app using the most recent environment.) I will update the code once this is done.